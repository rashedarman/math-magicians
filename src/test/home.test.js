import { render } from '@testing-library/react';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import HomePage from '../pages/Home';

it('Renders home page', () => {
  const result = render(
    <BrowserRouter>
      <HomePage />
    </BrowserRouter>,
  );
  expect(result).toMatchSnapshot();
});

// import renderer from 'react-test-renderer';
