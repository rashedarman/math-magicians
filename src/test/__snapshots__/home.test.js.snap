// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders home page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="main-layout"
      >
        <header>
          <div
            class="container"
          >
            <nav>
              <h1>
                Math Magicians
              </h1>
              <ul>
                <li>
                  <a
                    href="/"
                  >
                    Home
                  </a>
                </li>
                <li>
                  <a
                    href="/calculator"
                  >
                    Calculator
                  </a>
                </li>
                <li>
                  <a
                    href="/quote"
                  >
                    Quote
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </header>
        <div
          class="children container"
        >
          <h1
            style="margin-bottom: 3rem;"
          >
            Welcome to our page!
          </h1>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto dolore atque illo perspiciatis animi velit iure consequatur amet quisquam veniam neque modi, alias nesciunt expedita et. Aspernatur alias debitis, adipisci omnis ratione, illum soluta voluptatem tempore eius laborum incidunt cupiditate neque aliquid sunt magnam nam porro enim est totam placeat vel voluptatum voluptate molestias! Ab esse possimus vitae maxime assumenda!
          </p>
          <br />
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi eaque repudiandae autem inventore quam voluptate quasi perspiciatis. Nam eveniet expedita laborum optio dolorem. Ea facilis, aliquid veritatis dignissimos placeat voluptates pariatur repellendus eligendi necessitatibus officia nostrum quibusdam nihil nulla et sunt quo aspernatur earum veniam laborum eaque! Dicta eum dignissimos eveniet. Unde laboriosam, esse a animi distinctio sunt eius, saepe recusandae ut natus eligendi asperiores magni aut amet cumque vitae repellat! Tempora architecto nam obcaecati alias corporis cum praesentium est.
          </p>
        </div>
        <footer
          class="copyrightText"
        >
          <p
            class="copyright-text"
          >
            Copyright © Math Magicians 2023. All rights reserved.
          </p>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="main-layout"
    >
      <header>
        <div
          class="container"
        >
          <nav>
            <h1>
              Math Magicians
            </h1>
            <ul>
              <li>
                <a
                  href="/"
                >
                  Home
                </a>
              </li>
              <li>
                <a
                  href="/calculator"
                >
                  Calculator
                </a>
              </li>
              <li>
                <a
                  href="/quote"
                >
                  Quote
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </header>
      <div
        class="children container"
      >
        <h1
          style="margin-bottom: 3rem;"
        >
          Welcome to our page!
        </h1>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto dolore atque illo perspiciatis animi velit iure consequatur amet quisquam veniam neque modi, alias nesciunt expedita et. Aspernatur alias debitis, adipisci omnis ratione, illum soluta voluptatem tempore eius laborum incidunt cupiditate neque aliquid sunt magnam nam porro enim est totam placeat vel voluptatum voluptate molestias! Ab esse possimus vitae maxime assumenda!
        </p>
        <br />
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi eaque repudiandae autem inventore quam voluptate quasi perspiciatis. Nam eveniet expedita laborum optio dolorem. Ea facilis, aliquid veritatis dignissimos placeat voluptates pariatur repellendus eligendi necessitatibus officia nostrum quibusdam nihil nulla et sunt quo aspernatur earum veniam laborum eaque! Dicta eum dignissimos eveniet. Unde laboriosam, esse a animi distinctio sunt eius, saepe recusandae ut natus eligendi asperiores magni aut amet cumque vitae repellat! Tempora architecto nam obcaecati alias corporis cum praesentium est.
        </p>
      </div>
      <footer
        class="copyrightText"
      >
        <p
          class="copyright-text"
        >
          Copyright © Math Magicians 2023. All rights reserved.
        </p>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
